packages/cli/internal/pkg/cli/workflow/input.go

```diff
@@ -5,15 +5,9 @@ import (
 )
 
 type inputKey = string
-type inputUrl = string
+type inputUrl = interface{}
```

Comment:
If we are saying that we support not only URLs, then may be we should rename the corresponding type?

Reply:
Makes sense, fixed.